pages/community/slackccugl.md

```diff
@@ -0,0 +1,84 @@
+* This Slack is not an official support channel. It is a place where the community hang out along with some members of the Kotlin team. For official support, please use the Kotlin Forums, where messages are also persisted.
+* This Slack is not the best place for reporting bugs either. For this, please use YouTrack.  
+* Find the right channel to ask your question. There are channels for pretty much every topic. For a list of some of the channels see below.
+* New channels should be focused around a well-known technology or area of interest. Create them sparingly. And before creating a new channel, make sure it doesn’t exist in any other shape or form. 
+* When you ask a question, please be patient. Don’t repeat it. Or at least not immediately.
+* Don’t split messages into multiple ones. Ask it all in a single message.
+* Use code blocks. Don’t paste code as plain text. Slack supports Kotlin markup. 
+* While it’s polite to say “Hi” or “I have a question” before asking something, it doesn’t scale and leads to noise. You can say hello in the same message you ask the question if you like.
+* Use threads. It allows both yourself and others to follow the conversations easily. 
+* Use emojis to show gratitude as opposed to a message. It reduces the noise and gets the message across.
+* Don’t use emojis to tell people they’re in the wrong place or asking the wrong question. If they’re on the wrong channel, point them to the right one. If their question is badly worded, help them correct it. Have empathy. 
+* Take into account cultural differences. As they say, what the English say isn’t what the English mean. Remember this both as the receiver and producer of messages.  
+* Don’t create integrations with Slack. Given this is running on the free tier, the number is limited and thus it won’t be approved.
+* @channel, @here and other forms of notifications are disabled, even if you own a channel (unfortunately Slack doesn’t allow fine-grained permissions).
+* Use the #meta channel if you’re unsure about how to do something.  
+
+## Common Channels
+
+This is a list of some of the most popular channels, as well as a sample of the types of channels available. 
+
+* #android. For Android generic topics
+* #beginner. For more beginner questions. Better ask here than in #general if you’re new to Kotlin
+* #communities. A place to discuss physical communities. Also search for cities/countries as there are often channels with these names (#australia), as well as languages (i.e. #spanish).
+* #coroutines. Questions about coroutines and async programming in Kotlin
+* #dagger. About the Dagger framework. 
+* #datascience. Using Kotlin for Data Science. 
+* #functional. Functional programming topics with Kotlin
+* #gradle. Working with Gradle and Kotlin Script Gradle
+* #intellij. Working with IntelliJ IDEA and Kotlin
```

Comment:
We can rephrase it as "Regarding the Kotlin plugin for IntelliJ IDEA" which is the purpose of the channel. Or alternatively remove it, but the channel is there.